_docs-v4/event-model/event-object.md

```diff
@@ -169,6 +169,20 @@ A reference to the [Event Source](event-source-object) this event came from. If
 </td>
 </tr>
 
+<tr>
+<th>resourceId</th>
+<td markdown='1'>
+String. The `id` of a [Resource](resource-object) that this event is associated with. (**requires premium**)
```

Comment:
for the "requires premium" text, can you link to https://fullcalendar.io/docs/premium ?

Reply:
done